src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java

```diff
@@ -77,4 +89,14 @@ public static Boolean isBenefitTypeValid(String field) {
         }
         return false;
     }
+
+    public static String getLongBenefitNameDescriptionWithOptionalAcronym(String code, boolean isEnglish) {
+        Benefit benefit = getBenefitByCode(code);
+
```

Comment:
Good to handle, if benefit == null as it is possible value to be returned by getBenefitByCode() and we get all benefits types in some application

Reply:
Isn't that handled in the getBenefitByCode method above? It throws an exception...

Reply:
Yes, true. I mixed up the methods.